DigitalLearningSolutions.Web/Views/CookieConsent/CookiePolicy.cshtml

```diff
@@ -0,0 +1,157 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@model CookieConsentViewModel;
+@{
+    ViewData["Title"] = "Digital Learning Solutions - Cookie policy";
+    ViewData["DoNotDisplayNavBar"] = true;
+}
+<form method="post" asp-action="CookiePolicy">
+    @*<input type="hidden" asp-for="UserConsent" />*@
+
+    <div class="nhsuk-width-container app-width-container">
+        <div class="nhsuk-grid-row">
+            <div class="nhsuk-grid-column-full">
+
+                <div class="nhsuk-u-reading-width">
+
+                    <h1>Cookie policy</h1>
+
+                    <h2>What are cookies?</h2>

```

Comment:
Is there a way we could move line 18 to line 121 to a config/text file in the solution root so that we can have custom content for each application (if we are moving to a nuget package)?

Reply:
Done